dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +58,123 @@ def run(self) -> int:
                 account=dbt_credentials.get("account", ""),
                 database=dbt_credentials.get("database", "dwh"),
             ).get_columns_from_table(model, schema)
-
-        self.process_model(model, columns_sql)
+        if columns_sql:
+            self.orchestrator_document_model(model, columns_sql)
         return 0
 
+    def change_model_description(self, content: Dict[str, Any], model_name: str) -> Dict[str, Any]:
+        """Updates the model description from a model.
+
+        Args:
+            content (Dict[str, Any]): content of the schema.yml.
+            model_name (str): model name to change the description from.
+            description (str): description of the model.
+
+        Returns:
+            Dict[str, Any]: with the schema.yml updated.
+        """
+        model_doc_payload: List[Mapping[str, Any]] = [
+            {
+                "type": "confirm",
+                "name": "wants_to_document_model",
+                "message": "Model Description: This is my previously documented model. Document?",
+                "default": True,
+            },
+            {
+                "type": "text",
+                "name": "model_description",
+                "message": "Please write down your description:",
+            },
+        ]
+        user_input = UserInputCollector("model", model_doc_payload).collect()
+        if user_input.get("wants_to_document_model", False):
+            for model in content.get("models", []):
+                if model["name"] == model_name:
+                    model["description"] = user_input["model_description"]
+        return content
+
+    def find_model_in_dbt(self, model_name: str) -> Tuple[Optional[Path], bool]:
+        """
+        Method to find a model name in the DBT depository.
+
+            - If we find the sql of the model but there is no schema we return the Path
+            and False (to create the schema).
+            - If we find the sql of the model and there is schema we return the Path and True.
+
+        Args:
+            model_name (str): model name to find in the DBT repository.
+
+        Returns:
+            Tuple[Optional[Path], bool]: with the path of the sql model
+            and the boolean indicating if the schema.yml exists.
+        """
+        for root, _, files in os.walk(self.repository_path):
+            for file in files:
+                # TODO: Check how to avoid using target to discriminate compiled files in DBT.
```

Comment:
OHHH I think I know what you mean by "target" now. You mean the target directory where the compiled files and stuff go is that right?
```suggestion
                # TODO: Check how to avoid using target to discriminate compiled files in dbt.
```

I'm curious what you mean by this, you're not happy with having to check if the target folder is not in your walk list.

Maybe a cleaner way would be to do that would be to exclude the target directory from your walk list?

something like:
```python
exclude_pattern = r"\/target\/"
for root, dir, files in os.walk(self.repository_path):
    files = [f for f in files if not re.match(excludes, f)]
```
and then you can iterate through the files and all of the files contained in `/target` will be out of your list. Since there can be at least twice as many files in there as there are in `/models` this could give you a nice speed bump.

Reply:
yes, exactly that is a beautiful way! I will change it.